tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "3000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "2000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "2000-01-01"}},
+        },
+        {
+            "ocid": "ocds-213czf-6",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "2000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "3000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "2000-01-01"}},
+        },
+        {
+            "ocid": "ocds-213czf-7",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "2000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "2000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "3000-01-01"}},
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.contracting_process_setup, transforms.contract_status],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][1]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][2]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][3]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][4]["summary"]["status"] != "closed"
+    assert output["contractingProcesses"][5]["summary"]["status"] != "closed"
+    assert output["contractingProcesses"][6]["summary"]["status"] != "closed"
+
+
+def test_procurment_process():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"procurementMethod": "method", "procurementMethodDetails": "details"},
+        }
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases),
+        "1",
+        transforms=[transforms.contracting_process_setup, transforms.procurement_process],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["tender"] == releases[0]["tender"]
+
+
+def test_number_of_tenderers():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"numberOfTenderers": 123},
+        }
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases),
+        "1",
+        transforms=[transforms.contracting_process_setup, transforms.number_of_tenderers],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["tender"]["numberOfTenderers"] == 123
+
+
+def test_location():
```

Comment:
Should this test also cover the follow inputs:

1. A contracting process with multiple locations in `planning/project/locations`
2. Multiple contracting processes with different locations in `planning/project/locations`

Reply:
Fixed in https://github.com/open-contracting/ocdskit/pull/125/commits/51a8d5351fe041fdb5438f21122c56be5a5d83a5